lib/descope/api/v1/management/user.rb

```diff
@@ -609,11 +677,11 @@ def user_compose_create_body(
               sso_app_ids:
             )
             body[:invite] = invite
-            body[:verifiedEmail] = verified_email unless verified_email.nil? || !verified_email.empty?
-            body[:verifiedPhone] = verified_phone unless verified_phone.nil? || !verified_phone.empty?
-            body[:inviteUrl] = invite_url unless invite_url.nil? || !invite_url.empty?
-            body[:sendMail] = send_mail unless send_mail.nil? || !send_mail.empty?
-            body[:sendSMS] = send_sms unless send_sms.nil? || !send_sms.empty?
+            body[:verifiedEmail] = verified_email unless verified_email.nil? || verified_email.empty?
```

Comment:
what is all those "!" ("not") changes?

Reply:
this was a bug, i want to add the param to the body if its not nil or empty

doing unless ! empty ==  is empty which i don't want here